### PR TITLE
Handle AWS parquet format

### PR DESCRIPTION
**Pull request by MayCisco**
https://github.com/obsrvbl/devops/issues/9009

Add support of AWS parquet format

### CODE REVIEW

File: flowlogs_reader/flowlogs_reader.py [hunk 1]
```diff
@@ -474,6 +488,27 @@ def _read_streams(self):
...
+                self._has_field(event_data, 'srcport'),
+                self._has_field(event_data, 'dstport'),
+                self._has_field(event_data, 'protocol'),
+            ]
+        )
+
     def _reader(self):
         for event_data in self._read_streams():
-            yield FlowRecord(event_data)
+            if self._has_required_fields(event_data):
```
Comment (bbayles) [inline flowlogs_reader/flowlogs_reader.py:513]: I think this ought to be up to the caller. That is, some applications will have different required fields, and should not automatically have their formats filtered out.

**Review by jmarkey — APPROVED** [review]

**Comment by MayCisco** [comment]
@jmarkey Thanks for approving it. How is `flowlogs-reader` released?

**Comment by jmarkey** [comment]
> @jmarkey Thanks for approving it. How is `flowlogs-reader` released?

I would update our version in `setup.cfg` [here](https://github.com/obsrvbl/flowlogs-reader/blob/master/setup.cfg#L17) and edit the [README](https://github.com/obsrvbl/flowlogs-reader/blob/master/README.md) if necessary. Then, we can publish a [new release](https://github.com/obsrvbl/flowlogs-reader/releases) with the latest changes.

**Comment by bbayles** [comment]
You'll also need Michael or me to push the released version to PyPI after you bump the version and merge.

**Comment by MayCisco** [comment]
I bumped the version and merged the PR. Could you @bbayles or Michael release it?

**Pull request merged**